### ISSUE TITLE
Can we have alternate steps for using "Boot_recovery" from USB stick instead of netboot?

### ISSUE THREAD

**Issue by estarz** [NetAppDocs/ontap-systems-upgrade#95]
### Page URL

https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade/upgrade-set-up-new-nodes.html

### Page title

Set up the new nodes

### Summary

Hi,

So NetApp IT makes it hard for us and lock down the OS on our laptops.
On NetApp image MacOS, the forced firewall settings include blocking ping/ICMP.
And that makes this procedure fail.

Netboot works just fine, you are able to setup a web server on your laptop.
But step 7, "If you are prompted to continue the procedure, enter y, and when prompted for the package, enter the URL of the image file:", is challenging.

After you have provided the URL, ONTAP makes a connectivity check. That unfortunately includes a ping/ICMP test.
And that is blocked in the laptop firewall.

You could of course argue this is an ONTAP problem - why use ping to test if HTTP works?
Or a NetApp IT problem - why restrict ICMP if needed by ONTAP?

Well. Both of those things are hard to impact. And there are alternatives to netboot.
For the last 2 years I have been using USB sticks with the right versions of ONTAP and perform "boot_recovery".
Today I did this for a controller upgrade and it works just fine.

Documenting the alternate steps would make this procedure easier for NetApp employees and potentially customers with similar firewall challenges.

Basically the steps are: Populate a FAT formatted USB stick with the ONTAP image
Insert stick in ONTAP
From LOADER> "boot_recovery 9XXXPY_q_image.tgz"
Then just hit enter for the version of ONTAP to install in image1 and image2.
Remove stick.

IMHO a lot simpler than netboot. And with a NetApp MacBook, better success rate.

Thanks,
Erik

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-pcarriga** [comment]
@estarz Hi Eric, thanks for taking the time to provide this feedback. I'll discuss it with the SMEs and update the documentation accordingly.

**Comment by netapp-pcarriga** [comment]
@estarz Hi Eric, we've added the option to use "Boot_recovery" from a USB stick instead of netboot to the the documentation:
[Set up the new nodes](https://docs.netapp.com/us-en/ontap-systems-upgrade/upgrade/upgrade-set-up-new-nodes.html).
Thanks again for helping us improve the customer documentation.
